ksql-engine/src/test/java/io/confluent/ksql/IntegrationTestHarness.java

```diff
@@ -174,7 +186,7 @@ public void stop() {
   }
 
   public void assertExpectedResults(Map<String, GenericRow> actualResult,
-                                           Map<String, GenericRow> expectedResult) {
+                                    Map<String, GenericRow> expectedResult) {
```

Comment:
I know not really part of this PR, but... this method can be replaced with `assertThat(actualResult, equalTo(expectedResult)`